pcap-linux.c

```diff
@@ -4042,9 +4044,9 @@ create_ring(pcap_t *handle, int *status)
 #ifdef HAVE_TPACKET2
 	case TPACKET_V2:
 #endif
-		/* Note that with large snapshot length (say 64K, which is
-		 * the default for recent versions of tcpdump, the value that
-		 * "-s 0" has given for a long time with tcpdump, and the
+		/* Note that with large snapshot length (say 256K, which is
```

Comment:
256K is also the current default for Wireshark/TShark/dumpcap, so the comment should reflect that.